queries/smithy/highlights.scm

```diff
@@ -0,0 +1,116 @@
+
+; Keywords
+
+[
+  "namespace"
+  "service"
+  "structure"
+  "operation"
+  "list"
+  "map"
+  "union"
+  "resource"
+  "set"
+  "metadata"
+  "apply"
+  "for"
+  "with"
+  "enum"
+  "intEnum"
+] @keyword
+
+(simple_type_name) @type.builtin
+"use" @include
+
+[
+  ":"
+  "."
+] @punctuation.delimiter
+
+
+[
+  "("
+  ")"
+  "["
+  "]"
+  "{"
+  "}"
+  "#"
+] @punctuation.bracket
+
+(node_keywords) @boolean
+(number) @number
+(quoted_text) @string
+[
+  (comment)
+  (documentation_comment)
+] @comment
+[
+  ("@")
+] @operator
+
+(use_statement
+  (absolute_root_shape_id
+    (namespace) @namespace
+    (identifier) @type
+  ))
+
+(trait
+  (shape_id) @attribute
+)
+
+(node_string_value
+  (shape_id) @type
+)
+
+; (trait_body) @local.scope
+
+(shape_member_kvp
+  (identifier) @variable
+  (shape_id) @type
+)
+(shape_member_elided
+  (identifier) @variable
+)
+(structure_shape_statement
+  (identifier) @type.definition
```

Comment:
I think `@type.definition` is a bit a confusing name. It is meant for type identifiers defined by `typedef` type aliases. Is this the same semantics here?

Reply:
they were incorrect, in my PR they've been addressed as types now